sleap_nn/training/model_trainer.py

```
@@ -813,554 +848,905 @@ def train(self):
-        img = torch.squeeze(img, dim=1).to(self.device)
-        output = self.model(img)
-        return {
-            "MultiInstanceConfmapsHead": output["MultiInstanceConfmapsHead"],
-            "PartAffinityFieldsHead": output["PartAffinityFieldsHead"],
-        }
+        def run_subprocess():
+            process = subprocess.Popen(
+                [
+                    "python",
+                    "-m",
+                    f"sleap_nn.training.get_bin_files",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)